src/common/WorkQueue.h

```diff
@@ -434,7 +434,7 @@ class ThreadPool : public md_config_obs_t {
   };
 private:
   vector<WorkQueue_*> work_queues;
-  int last_work_queue;
+  int next_work_queue;
```

Comment:
sorry not catch your idea clearly... this var is initialized in construction list, so what is your suggestion?